### DISCUSSION TITLE
v3.2.2补数失败

### DISCUSSION THREAD

**Discussion by renlixing** [apache/dolphinscheduler#16614] [Q&A]
我安装了3.2.2版本，运行时选择补数，可是只有第一个日期可以运行成功，运行第二个日期就会报错租户不存在。我在运行时选择了租户(否则第一个日期也不会成功)。
这是运行时的参数配置：
<img width="904" alt="bs" src="https://github.com/user-attachments/assets/3960266b-ebab-47bd-976f-398f37f6e23c">
以下是运行的结果（设置重试次数为1，所以运行第二个日期会有2个失败的test_shell）
<img width="1064" alt="Snipaste_2024-09-13_13-23-10" src="https://github.com/user-attachments/assets/ae3ea05e-a93d-43b7-9587-5d6824716dd0">
<img width="1037" alt="Snipaste_2024-09-13_13-23-38" src="https://github.com/user-attachments/assets/13c24186-92bf-4ff2-baf9-0b24fa67d07b">
这是shell报错日志
<img width="1053" alt="Snipaste_2024-09-13_13-21-35" src="https://github.com/user-attachments/assets/50581784-e8cd-4e85-bdc6-9f025bbd99cb">

**Comment by gwdgithubnom** [comment]
这是因为机器上没有default的用户，tenantcode对应的是机器用户标识

**Reply by taincheng** [reply]
这个是 3.2.1的老bug 了，开启多天的补数任务，第二个任务就会使用 default去执行

**Comment by renlixing** [comment]
是，换版本了



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年2月28日(周五) 下午4:02
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [apache/dolphinscheduler] v3.2.2补数失败 (Discussion #16614)




 
这个是 3.2.1的老bug 了，开启多天的补数任务，第二个任务就会使用 default去执行

**Reply by gaoxingxs** [reply]
> 是，换版本了
> […](#)
> ---原始邮件--- 发件人: ***@***.***&gt; 发送时间: 2025年2月28日(周五) 下午4:02 收件人: ***@***.***&gt;; 抄送: ***@***.******@***.***&gt;; 主题: Re: [apache/dolphinscheduler] v3.2.2补数失败 (Discussion #16614) 这个是 3.2.1的老bug 了，开启多天的补数任务，第二个任务就会使用 default去执行 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

请问换到了哪个版本？

**Comment by qctc-sunyu** [comment]
我在3.2.2上修复了这个问题, 但是3.3.0 代码有很大重构, 这部分代码已经没有, 还可以在3.2.2上提交Fix吗?

**Reply by Sud0x67** [reply]
我也修了一个bug，不知道还出3.2.2之后的bug fix 版本不出了
